src/checkbox/ux-checkbox.ts

```diff
@@ -1,4 +1,5 @@
 import { customElement, bindable, ViewResources, View, processAttributes } from 'aurelia-templating';
+import { computedFrom } from 'aurelia-framework';
```

Comment:
That makes sense. I didn't know you can import that from aurelia-binding.  Thanks, will get this adjusted shortly.